types/foundry-vtt-api/types/framework/entities/item.d.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * The Collection of Item entities
+ * The items collection is accessible within the game as game.items
+ */
+declare class Items extends Collection<Item> {
+    entities: Item[];
+
+    values(): IterableIterator<Item>;
+
+    /* -------------------------------------------- */
+    /*  Collection Properties                       */
+    /* -------------------------------------------- */
+
+    static get instance(): Items;
+
+    /**
+     * Elements of the Items collection are instances of the Item class, or a subclass thereof
+     */
+    get object(): Item;
+
+    /* -------------------------------------------- */
+    /*  Collection Management Methods               */
+    /* -------------------------------------------- */
+
+    insert(entity: Item): void;
+
+    get(id: string, { strict }?: { strict?: boolean }): Item;
+
+    /**
+     * Register an Item sheet class as a candidate which can be used to display Items of a given type
+     * See EntitySheetConfig.registerSheet for details
+     */
+    static registerSheet(...args: any): void;
+
+    /**
+     * Unregister an Item sheet class, removing it from the list of avaliable sheet Applications to use
+     * See EntitySheetConfig.unregisterSheet for details
+     */
+    static unregisterSheet(...args: any): void;
+
+    /**
+     * Return an Array of currently registered sheet classes for this Entity type
+     */
+    static get registeredSheets(): any[];
+}
+
+declare class Item extends Entity {
+    /**
+     * Configure the attributes of the ChatMessage Entity
+     *
+     * @returns baseEntity			The parent class which directly inherits from the Entity interface.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/framework/entities/item.d.ts;
  Line: 51;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.